Source/SpiderEye.Windows/Interop/KeyMapper.cs

```diff
@@ -126,6 +126,7 @@ private static Keys MapKey(Key key)
             { Key.Insert, Keys.Insert },
             { Key.Delete, Keys.Delete },
             { Key.QuestionMark, Keys.OemQuestion },
+            { Key.Comma, Keys.Oemcomma },
```

Comment:
Sadly, this is display as `CTRL+Oemcomma`.

We could implement a workaround in `WinFormsLabelMenuItem`, by setting the `menuItem.ShortcutKeyDisplayString` to `CTRL+,`. But this wouldn't work as well, since `CTRL` should be translated to `Strg` on german platforms...

We would have to use the `KeysConverter` to convert the `CTRL` key to a string and the append the `+,`